pkg/controller/direct/batch/mapper.generated.go

```diff
@@ -96,9 +96,7 @@ func AllocationPolicy_Disk_ToProto(mapCtx *direct.MapContext, in *krm.Allocation
 		return nil
 	}
 	out := &pb.AllocationPolicy_Disk{}
-	if oneof := AllocationPolicy_Disk_Image_ToProto(mapCtx, in.Image); oneof != nil {
-		out.DataSource = oneof
-	}
+	// MISSING: Image
```

Comment:
I meant to rename the go field but not the json tag, because I think this trick lets the generated mappers work correctly.